SPECS/php/php-8.3.13-phpinfo.patch

```diff
@@ -25,7 +25,7 @@ diff -up ./ext/standard/tests/general_functions/phpinfo.phpt.phpinfo ./ext/stand
 @@ -17,7 +17,6 @@ PHP Version => %s
  
  System => %s
- Build Date => %s%a
+ Build Date => %r(.+?)%r
```

Comment:
Why are we making this change?

Reply:
%s%a is looser and matches anything, even there is empty space after Build Date, To ensure stricter vallidation %r(.+?)%r is used 
Reference: https://git.remirepo.net/cgit/rpms/php/php83.git/commit/?id=b6287db42acbda84eae51427db158db8828e7f5d